### DISCUSSION TITLE
당근 테크밋업 발표를 하고 (발표를 앞둔 모든 분께) - 정현수 기술 블로그

### DISCUSSION THREAD

**Discussion by giscus** [junghyeonsu/junghyeonsu.dev.comment#63] [Announcements]
# 당근 테크밋업 발표를 하고 (발표를 앞둔 모든 분께) - 정현수 기술 블로그

당근 테크밋업 "아이콘, 브랜딩 에셋까지 디자인시스템으로" 발표하고 나서 배운 점과 발표를 앞둔 사람들에게

https://junghyeonsu.com/posts/daangn-tech-meetup-retrospect/

**Comment by ainochi-kor** [comment]
발표를 하게 된다면 한번쯤 읽어보면 정말 좋은 글이네요! 발표 중에도 스스로에게 피드백도 하게 되고, 발표 하는 분들께 소개해줘야겠어요ㅎㅎ
발표도 그렇고 이 글도 그렇고 꼭 누군가에게는 큰 도움이 되었을거라고 생각합니다! 
발표 너무 고생하셨구 블로그도 멋지네요ㅎㅎ 다음 포스트도 기대할게요!

**Comment by endmoseung** [comment]
내용도 내용인데 발표 장표 퀄리티가 미쳤어요

**Comment by rotoshine** [comment]
고생하셨어요~
